racun-service/src/main/java/rs/edu/raf/banka/racun/controller/UgovorController.java

```diff
@@ -124,42 +126,32 @@ public ResponseEntity<?> modifyUgovorDocument(@RequestHeader("Authorization") St
 
     @PostMapping(value = "/stavka", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<?> createStavka(@RequestHeader("Authorization") String token, @RequestBody TransakcionaStavkaCreateRequest request) {
-        try
-        {
-            var stavka = ugovorService.addStavka(request);
+        try {
+            var stavka = ugovorService.addStavka(request, token);
```

Comment:
Stavku mogu da kreiraju i agenti i supervizori kod svih ugovora tako da ne treba token.

Reply:
Opet potreban je agentId u slucaju ako nije supervizor

Reply:
Koliko sam razumeo, stavku u ugovoru kreira onaj korisnik koji je kreirao i ugovor tako da token ne treba. Eventualno moze ostati radi neke druge provere, a to je da li je taj ulogovani korisnik upravo korisnik koji je kreirao taj ugovor za koji se kreira stavka. Ja bih ostavio token i onda u servisu proverio da li je id ulogovanog korisnika jednak id-u korisnika koji je kreirao ugovor za koji se kreira stavka.

Reply:
> Eventualno moze ostati radi neke druge provere, a to je da li je taj ulogovani korisnik upravo korisnik koji je kreirao taj ugovor za koji se kreira stavka. Ja bih ostavio token i onda u servisu proverio da li je id ulogovanog korisnika jednak id-u korisnika koji je kreirao ugovor za koji se kreira stavka.

Ovo ^^

Ja bih sačuvao koje kreirao stavku, čisto da se zna, ali glavna provera je da li je supervizor, ili ako je agent, da je to agent koji je napravio ugovor.